### DISCUSSION TITLE
Offline mode

### DISCUSSION THREAD

**Discussion by DeadNumbers** [nicolargo/glances#3039] [General]
I prefer to strictly filter my traffic and block unnecessary connections in the firewall. However, if I block traffic from glances, the monitor start after http timeout. Please make an offline mode that would not try to connect to 169.254.169.254 or pypi. Thanks!

**Comment by nicolargo** [comment]
Hi @DeadNumbers 

Use the following to disable plugins talking with Internet:

```
[global]
# Does Glances should check if a newer version is available on PyPI ?
check_update=False

[cloud]
# Documentation: https://glances.readthedocs.io/en/latest/aoa/cloud.html
# This plugin is disabled by default
disable=True
```

**Reply by DeadNumbers** [reply]
Thanks!
